src/index.js

```diff
@@ -9,47 +9,78 @@ import {
   getThingAll,
   getStringNoLocale,
   removeThing,
-  FetchError
+  getPodUrlAll,
 } from "@inrupt/solid-client";
 
 import {
   login,
   handleIncomingRedirect,
   getDefaultSession,
-  fetch
+  fetch,
 } from "@inrupt/solid-client-authn-browser";
 
 import { SCHEMA_INRUPT, RDF, AS } from "@inrupt/vocab-common-rdf";
 
+const podUrlInput = document.getElementById("PodURL");
+const statusText = document.getElementById("labelStatus");
 const buttonLogin = document.querySelector("#btnLogin");
 const buttonCreate = document.querySelector("#btnCreate");
-buttonCreate.disabled=true;
+buttonCreate.setAttribute("disabled", "disabled");
+
 const labelCreateStatus = document.querySelector("#labelCreateStatus");
+const availablePods = document.querySelector("#availablePods");
 
 // 1a. Start Login Process. Call login() function.
 function startLogin() {
   return login({
-
     oidcIssuer: "https://broker.pod.inrupt.com",
 
     redirectUrl: window.location.href,
-    clientName: "Getting started app"
+    clientName: "Getting started app",
   });
 }
 
 // 1b. Login Redirect.
 // When redirected after login, call handleIncomingRedirect() function to
 // finish the login process by retrieving session information.
 async function finishLogin() {
-    await handleIncomingRedirect();
-    const session = getDefaultSession();
-    if (session.info.isLoggedIn) {
+  await handleIncomingRedirect();
+  const session = getDefaultSession();
+  if (session.info.isLoggedIn) {
+    buttonLogin.setAttribute("disabled", "disabled");
+
+    try {
+      const pods = await getPodUrlAll(session.info.webId, {
+        // This isn't strictly needed, as the WebID must be a public resource,
+        // but for now it's safest to include it:
+        fetch: fetch,
+      });
+
+      const availablePodsList = document.createElement("ul");
+
+      pods.forEach((url) => {
+        const listItem = document.createElement("li");
+        listItem.textContent = url;
+        availablePodsList.appendChild(listItem);
+      });
+
+      availablePods.appendChild(availablePodsList);
+      // Show the list:
+      availablePods.removeAttribute("style");
+
+      // Set the input value to the first pod found, if any, stripping the trailing slash:
+      podUrlInput.value = pods.length > 0 ? pods.at(0).replace(/\/$/, "") : "";
+
       // Update the page with the status.
-      document.getElementById("labelStatus").textContent = `Logged in with WebID ${session.info.webId}`;
-      document.getElementById("labelStatus").setAttribute("role", "alert");
+      statusText.textContent = `Logged in with WebID ${session.info.webId}`;
```

Comment:
We have a **_general_** (i.e., there's always justifiable exceptions (if extremely rare, in my experience)) Inrupt coding convention (regardless of programming language) to always delimit all variables with '[' and ']' (main reason is to easily spot pesky instances of leading and/or trailling whitespace, but also to consistently denote the notion of '[here is a value that is variable in some sense]'): i.e.,
```suggestion
      statusText.textContent = `Logged in with WebID [${session.info.webId}]`;
```